modules/ossm-rn-fixed-issues.adoc

```diff
@@ -15,7 +15,22 @@ Provide the following info for each issue if possible:
 ////
 
 The following issues have been resolved in the current release:
-//current release is 2.5.1/2.4.7/2.3.11 scheduled for April 18, 2024
+
+//current release is 2.5.2/2.4.8/2.3.12 scheduled for May 22, 2024
+
+* https://issues.redhat.com/browse/OSSM-6331[OSSM-6331] Previously, invalid values were accepted in `componentLevels` map `.spec.general.logging.componentLevels` and the `ServiceMeshControlPlane` was created. Now, an error message is shown if an invalid value is used, and the control plane is not created.
+
+* https://issues.redhat.com/browse/OSSM-6290[OSSM-6290] Previously, the **Project** filter of the **Istio Config** list page did not work correctly. All `istio` items are displayed even if you select a specific project from the dropdown. Now, only the Istio items belonging to the selected project in the filter dropdown are displayed.
```

Comment:
Two comments here:
- 'istio items' are displayed in two different ways in the sentence: `istio` items and Istio items. I think the name style should be consistent along the sentence. Besides, this is related to istio config list page, so maybe it is better to talk about 'istio config items' instead of just 'istio items'.
- All `istio` items are displayed... it means from all namespaces. Not sure if it is clear enough here.